sormas-app/app/src/main/java/de/symeda/sormas/app/sample/edit/SampleEditFragment.java

```diff
@@ -244,6 +260,17 @@ public void onChange(ControlPropertyField field) {
         }
     }
 
+    void showLabExternal(FragmentSampleEditLayoutBinding contentBinding, int show){
```

Comment:
This method should have a more general name, e.g. setVisibilitiesBasedOnLabType or something like that; replace the "show" parameter with the actual lab type and set visibilities in a switch statement